openwisp_radius/tasks.py

```diff
@@ -39,3 +49,38 @@ def delete_unverified_users(older_than_days=1, exclude_methods=''):
 @shared_task
 def convert_called_station_id(unique_id=None):
     management.call_command('convert_called_station_id', unique_id=unique_id)
+
+
+@shared_task
+def send_login_email(accounting_data):
+    User = get_user_model()
+    username = accounting_data.get('username', None)
+    org_uuid = accounting_data.get('organization')
+    user = User.objects.filter(username=username).first()
+    if user:
+        organization = (
+            user.openwisp_users_organization.all().filter(id=org_uuid).first()
+        )
+        if organization:
+            from sesame.utils import get_query_string
+
+            org_radius_settings = organization.radius_settings
+            login_url = org_radius_settings.login_url
+            if login_url:
+                activate(user.language)
+                one_time_login_url = login_url + get_query_string(user)
+                subject = _('New radius accounting session started')
+                context = {
+                    'user': user,
+                    'subject': subject,
+                    'call_to_action_url': one_time_login_url,
+                    'call_to_action_text': _('Manage Session'),
+                }
+                body_html = loader.render_to_string(
+                    'radius_accounting_start.html', context
+                )
```

Comment:
use django's  strip_tags to prepare the textual message (`body_text`)

Reply:
Strip tag is already used for `body_text` in the send_mail function of openwisp_utils so if I pass the HTML then it strips the tag by default if the email type is text.